spread.yaml

```diff
@@ -52,12 +52,12 @@ backends:
 
   lxd-nested:
     type: lxd
-    container-profiles: core24
+    container-profiles: core26
     systems:
       - ubuntu-24.04:
-          image: ucspread24
+          image: ucspread26
```

Comment:
this looks odd - the system is ubuntu-24.04? Maybe add a TODO that we need to add a new entry once an image exists?